Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.3.1
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential nodejs && apt-get clean
```

Comment:
I think it's better if the `upgrade` and the `install` steps are separate so the upgraded image can be cached.